NitroxPatcher/Patcher.cs

```diff
@@ -72,7 +72,6 @@ public static void Restore()
 
         foreach (IDynamicPatch patch in container.Resolve<IDynamicPatch[]>())
         {
-            Log.Debug($"Restoring dynamic patch {patch.GetType().Name}");
```

Comment:
We should keep that debug thing if we wanna make sure unpatching is actually working